### PR TITLE
Changes Amazon

### PR DESCRIPTION
**Pull request by gallegonovato**

### CODE REVIEW

**Review by nickspaargaren — CHANGES_REQUESTED** [review]

File: amazon.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 _domainkey.amazonaws.com
+cdn.jsdelivr.net
```
[the remark this answers is not in the document]
Comment (gallegonovato) [reply amazon.txt:2]: Done

File: amazon.txt [hunk 2]
```diff
@@ -2454,4 +2845,7 @@ dl1d2m8ri9v3j.cloudfront.net
 dz2fz0bgyq9tn.cloudfront.net
 dzpu6za66svjl.cloudfront.net
 server-204-246-164-0.sin2.r.cloudfront.net
-server-204-246-168-0.jfk1.r.cloudfront.net
\ No newline at end of file
+server-204-246-168-0.jfk1.r.cloudfront.net
+amazon.com
```
[the remark this answers is not in the document]
Comment (gallegonovato) [reply amazon.txt]: Done

File: amazon.txt [hunk 3]
```diff
@@ -307,6 +307,24 @@ aax-us.amazon-adsystem.com
...
 aax.amazon.com
+ams-pageview-public.s3.amazonaws.com
+amazon-adsystem.com
+z-na.amazon-adsystem.com
+c.amazon-adsystem.com
+mobileanalytics.us-east-1.amazonaws.com
+spyhunter-download.s3.amazonaws.com
+wp-stat.s3.amazonasw.com
+amazon-france.xyz
+amazeamazon.ddns.net
```
Comment (nickspaargaren) [inline amazon.txt:318]:
```suggestion
```

This is not a Amazon domain

File: amazon.txt [hunk 4]
```diff
@@ -307,6 +307,24 @@ aax-us.amazon-adsystem.com
...
+yourmailamazon99.redirectme.net
+amazon34.duckdns.org
```
Comment (nickspaargaren) [inline amazon.txt:320]:
```suggestion
```

This is not a Amazon domain

**Review by nickspaargaren — CHANGES_REQUESTED** [review]
After these changes we can merge them!

File: amazon.txt (hunk 4 above)
Comment (gallegonovato) [reply amazon.txt]: Delete

File: amazon.txt (hunk 3 above)
Comment (gallegonovato) [reply amazon.txt]: Delete

File: amazon.txt [hunk 5]
```diff
@@ -307,6 +307,22 @@ aax-us.amazon-adsystem.com
...
 aax.amazon.com
+ams-pageview-public.s3.amazonaws.com
+amazon-adsystem.com
+z-na.amazon-adsystem.com
+c.amazon-adsystem.com
+mobileanalytics.us-east-1.amazonaws.com
+spyhunter-download.s3.amazonaws.com
+wp-stat.s3.amazonasw.com
+amazon-france.xyz
+yourmailamazon99.redirectme.net
```
Comment (nickspaargaren) [inline amazon.txt:318]:
```suggestion
```

**Review by nickspaargaren — CHANGES_REQUESTED** [review]

**Comment by gallegonovato** [comment]
Good morning:

Changes made. :)

I don't know what you think about making a GAFAM , of a hosts to avoid the Windows stuff like https://github.com/nickspaargaren/no-amazon/issues/4 comments.

Best regards

File: amazon.txt (hunk 5 above)
Comment (gallegonovato) [reply amazon.txt]: Delete

**Review by nickspaargaren — APPROVED** [review]

**Pull request merged**